app/models/api/v1/user_registration.rb

```diff
@@ -10,6 +10,7 @@ class API::V1::UserRegistration
   attribute :password,              String
   attribute :password_confirmation, String
   attribute :email,                 String
+  attribute :email_subscriber,      Boolean, :default => false
```

Comment:
This flag, email_subscriber, might be slightly better named. What it ultimately means is, "is this user subscribed to enews?". So, something like enews_subscribed might read a little bit better. It also makes it a little more parallel to the "subscribed" state of things in other parts of the code. Very minor.

Reply:
@DALove1025 @scytacki I went ahead and changed email_subscriber to email_subscribed. I also changed email_subscriber to email_subscribed in portal-pages https://github.com/concord-consortium/portal-pages/pull/57